### PR TITLE
Add upcoming by-elections to the home page

### PR DESCRIPTION
**Pull request by symroe**
screen shot 2017-11-29 at 10 59 45

### CODE REVIEW

**Comment by symroe** [comment]
[Add upcoming elections to Who home page](https://trello.com/c/gjwYTQtw/1081-add-upcoming-elections-to-who-home-page)

File: wcivf/apps/elections/models.py [hunk 1]
```diff
@@ -145,16 +145,20 @@ class Post(models.Model):
...
 class PostElection(models.Model):
     post = models.ForeignKey(Post)
     election = models.ForeignKey(Election)
     contested = models.BooleanField(default=True)
 
+
+    def friendly_name(self):
+        # TODO clearly not every election is a by-election
+        # This will get better once we import more info from YNR/EE
+        return "{} ward by-election".format(
```
Comment (chris48s) [inline wcivf/apps/elections/models.py]: Given we are going to have to deal with a Welsh Assembly by-election soon (before Feb 7th - source: https://www.itv.com/news/wales/update/2017-11-20/alyn-deeside-vacant-formal-declaration/ ), lets not hard-code "ward" here. If we can pick up the post/geography type from the `Post` model, can we put in a special case? If not, lets stick to `return "{} by-election".format(` here.

**Comment by chris48s** [comment]
refs #235

**Comment by symroe** [comment]
Cool, fixup pushed that should deal with the one case we know about, we can add more if we need to, but obviously I hope we'll have fixed it properly before then

**Pull request merged**